mobile-ys/src/app/pages/home/home.page.ts

```diff
@@ -41,4 +46,7 @@ export class HomePage implements OnInit, OnDestroy {
             })
         )
     }
+    getClient():void{
```

Comment:
The users Must can see the products on the home page without log in. Fix the interceptor service, remove navigation to Login.

Reply:
Okey, i'll see it.